examples/api-dotenv-request/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog for `api-dotenv-request`
```

Comment:
@juanfcarrillo Can we rename the example to `api-request`? I don't think it's necessary to add `-dotenv-` since that's the name of the root project.